pages/web-literacy.jsx

```diff
@@ -0,0 +1,158 @@
+var React = require('react');
+
+var Illustration = require('../components/illustration.jsx');
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
+
+var WebMaps = React.createClass({
+  render: function() {
+    return (
+      <div className="row web-maps">{this.props.children}</div>
+    );
+  }
+});
+
+var WebMap = React.createClass({
+  render: function() {
+    return (
+      <div className="web-map col-sm-4 col-md-4 col-lg-4">
+        <h2>{this.props.head}</h2>
+        <span>{this.props.subhead}</span>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+
+var WebMapItem = React.createClass({
+  render: function() {
+    return (
+      <div className="web-map-item">{this.props.head}</div>
+    );
+  }
+});
+
+var ActivitiesPage = React.createClass({
+  statics: {
+    pageClassName: "web-literacy"
+  },
+  render: function() {
+    return (
+      <div>
+        <section>
+          <h1>Web Literacy</h1>
+          <Illustration
+            src1x="/img/web-literacy-page/icon-web-literacy.svg"
+            alt="web literacy"
+          >
+            <h2>
+              Web Literacy is the skills and competencies needed for reading, writing, and participating on the web.
```

Comment:
You'll likely find more of these, I'll make a pass over the whole file closer to the end and clean them all up.